static/js/components/ExpandedLearningResourceDisplay.js

```diff
@@ -66,159 +82,168 @@ const ExpandedLearningResourceDisplay = (props: Props) => {
+          {isCoursewareResource(object.object_type) && url ? (
+            <div className="course-links">
+              <div>
+                <a
+                  className="link-button"
+                  href={url}
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  {`Take ${capitalize(object.object_type)}`}
+                  {offeredBy && object.object_type !== LR_TYPE_BOOTCAMP
+                    ? ` on ${offeredBy}`
+                    : null}
+                </a>
+              </div>
             </div>
+          ) : null}
+          <div className="course-title">{object.title}</div>
+          <div className="course-description">
+            <TruncatedText
+              text={entities.decode(striptags(object.short_description))}
+              lines={5}
+              estCharsPerLine={100}
+              showExpansionControls
+            />
           </div>
-        ) : null}
-        {object.object_type === LR_TYPE_VIDEO ? (
-          <React.Fragment>
-            {// $FlowFixMe: only videos will get to this code
-              object.duration ? (
-                <div className="course-info-row">
-                  <i className="material-icons restore">restore</i>
-                  <div className="course-info-label">Duration:</div>
-                  <div className="course-info-value">
-                    {// $FlowFixMe: only videos will get to this code
-                      formatDurationClockTime(object.duration)}
+          {!emptyOrNil(object.topics) ? (
+            <React.Fragment>
+              <div className="course-subheader row">Topics</div>
+              <div className="course-topics">
+                {object.topics.map((topic, i) => (
+                  <div className="grey-surround facet" key={i}>
+                    {topic.name}
                   </div>
-                </div>
-              ) : null}
-            {offeredBy ? (
-              <div className="course-info-row">
-                <i className="material-icons local_offer">local_offer</i>
-                <div className="course-info-label">Offered By:</div>
-                <div className="course-info-value">{offeredBy}</div>
+                ))}
               </div>
+            </React.Fragment>
+          ) : null}
+          <div className="course-subheader row">Info</div>
+          {cost ? courseInfoRow("attach_money", "Cost:", cost) : null}
+          {selectedRun && selectedRun.level
+            ? courseInfoRow("bar_chart", "Level:", selectedRun.level)
+            : null}
+          {!emptyOrNil(instructors)
+            ? courseInfoRow("school", "Instructors:", R.join(", ", instructors))
+            : null}
+          {object.object_type === LR_TYPE_PROGRAM
+            ? courseInfoRow(
+              "menu_book",
+              "Number of Courses:",
+              object.items.length
+            )
```

Comment:
Maybe a question for @abdulkdawson - since the items in each list/path will now be shown, should the same be done for programs? Show a list of courses instead of just the number of courses?

Reply:
Going to assume this will be a separate PR if it needs to change from current behavior